extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -48,6 +48,8 @@ tags: {  # <amp-story>
       allowed_protocol: "http"
       allowed_protocol: "https"
     }
+    deprecation: "AMP-STORY-BOOKEND"
+    deprecation_url: "https://www.ampproject.org/docs/reference/components/amp-story#announcements"
```

Comment:
⚠️ Heads up: Your #15465 changes the name of this section, which will change the URL fragment as well.

Reply:
Done.